src/components/home-container/index.jsx

```diff
@@ -25,7 +25,7 @@ function HomeContainer() {
         {isNewTaskRequested && (
           <CreateTask
             onSuccessfullTaskEntry={() => setIsNewTaskRequested(false)}
-            onDeleteBtnClick={() => setIsNewTaskRequested(false)}
+            onDelete={() => setIsNewTaskRequested(false)}
```

Comment:
Using arrow functions in render is a bad idea because “by design” renders occur at high volume during a component’s lifecycle; allocating a new pointer for every arrow function. Abstaining from arrow functions in render will ensure you’re keeping your component’s memory usage on a diet.

// see https://www.digitalocean.com/community/tutorials/react-new-way-to-handle-events

```
function onDeleteTask() {
      setIsNewTaskRequested(false)
}
```

```suggestion
            onDelete={onDeleteTask}
```